exercises/Python_Principles_4.ipynb

```diff
@@ -281,31 +253,20 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Here the expression `b > a` is evaluated, which is `False` so instead of the code below the if statement being run Python goes to the else statement. The code below the else keyword is then run, therefore `a is greater than b` is printed."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "In the cell below, compare two variables using operators to form an expression and print out different statements based upon those conditions. "
+    "Write below a code which checks if `a` is negative and print the statement whether `a` is negative of positive"
```

Comment:
@AlexTeslenko what do you think of this change `Write below some code which checks` instead of `Write below a code` ?

Reply:
yep, agree